acceptance-tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Stories/ServiceSearchTests.cs

```diff
@@ -1,95 +1,183 @@
+        string radius, string postcode, string statusCode, string searchTriggerEventId, string serviceSearchTypeId,
+        HttpStatusCode expectedStatusCode)
+    {
+        this.Given(s =>
+                _steps.GivenIHaveASearchServiceRequest(radius, postcode, statusCode, searchTriggerEventId,
+                    serviceSearchTypeId))
+            .When(s => _steps.WhenISendARequest())
+            .Then(s => _steps.ThenExpectedStatusCodeReturned(expectedStatusCode))
+            .BDDfy();
+    }
+
+    [Theory]
+    [InlineData("15", "E1 2EN", "200", "1", "1", HttpStatusCode.OK)]
+    public void
+        Service_Search_Metrics_Endpoint_Returns_A_200_When_An_Initial_Postcode_Search_Is_Done_As_A_Connect_User(
+            string radius, string postcode, string statusCode, string searchTriggerEventId, string serviceSearchTypeId,
+            HttpStatusCode expectedStatusCode)
+    {
+        this.Given(s =>
+                _steps.GivenIHaveASearchServiceRequest(radius, postcode, statusCode, searchTriggerEventId,
+                    serviceSearchTypeId))
+            .When(s => _steps.WhenISendARequest())
+            .Then(s => _steps.ThenExpectedStatusCodeReturned(expectedStatusCode))
+            .BDDfy();
+    }
+
+    [Theory]
+    [InlineData("15", "E1 2EN", "200", "2", "2", HttpStatusCode.OK)]
+    public void
+        Service_Search_Metrics_Endpoint_Returns_A_200_When_A_Subsequent_Filter_Postcode_Search_Is_Done_As_A_Find_User(
+            string radius, string postcode, string statusCode, string searchTriggerEventId, string serviceSearchTypeId,
+            HttpStatusCode expectedStatusCode)
+    {
+        this.Given(s =>
+                _steps.GivenIHaveASearchServiceRequest(radius, postcode, statusCode, searchTriggerEventId,
+                    serviceSearchTypeId))
+            .When(s => _steps.WhenISendARequest())
+            .Then(s => _steps.ThenExpectedStatusCodeReturned(expectedStatusCode))
+            .BDDfy();
+    }
+
+    [Theory]
+    [InlineData("15", "E1 2EN", "200", "2", "1", HttpStatusCode.OK)]
+    public void
+        Service_Search_Metrics_Endpoint_Returns_A_200_When_A_Subsequent_Filter_Postcode_Search_Is_Done_As_A_Connect_User(
+            string radius, string postcode, string statusCode, string searchTriggerEventId, string serviceSearchTypeId,
+            HttpStatusCode expectedStatusCode)
+    {
+        this.Given(s =>
+                _steps.GivenIHaveASearchServiceRequest(radius, postcode, statusCode, searchTriggerEventId,
+                    serviceSearchTypeId))
+            .When(s => _steps.WhenISendARequest())
+            .Then(s => _steps.ThenExpectedStatusCodeReturned(expectedStatusCode))
+            .BDDfy();
+    }
+
+    [Theory]
+    [InlineData("15", "E1 2EN", "300", "1", "2", HttpStatusCode.OK)]
+    public void
+        Service_Search_Metrics_Endpoint_Returns_A_200_For_An_Initial_Postcode_Search_As_A_Find_User_After_the_Service_Search_API_Responds_With_A_Non_200_Response_Code(
```

Comment:
(As discussed on a call)

refactor: From the perspective of POSTing to the `/api/metrics/service-search` endpoint I think it's better to include a couple of these into your 200 status code test block instead of defining them as separate tests